src/get-started/uninstall/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Uninstall Flutter
+description: How to remove the Flutter SDK.
+toc: true
+os-list: [Windows, macOS, Linux, ChromeOS]
+---
+
+To remove all of Flutter from your {{os}} development machine,
+delete the directories that store Flutter and its configuration files.
+
+{% comment %} Nav tabs {% endcomment -%}
+<ul class="nav nav-tabs" id="base-os-tabs" role="tablist">
+{% for os in page.os-list %}
+{% assign id = os | downcase -%}
+  <li class="nav-item">
+    <a class="nav-link {%- if id == 'windows' %} active {% endif %}" id="{{id}}-tab" href="#{{id}}" role="tab" aria-controls="{{id}} {{id}}-dl {{id}}-pub" aria-selected="true">{{os}}</a>
+  </li>
+{% endfor -%}
+</ul>
+
+{% comment %} Tab panes {% endcomment -%}
+<div class="tab-content">
+{% for os in page.os-list %}
+{% assign id = os | downcase -%}
+
+{% case os %}
+{% when 'Windows' -%}
+{% assign dirinstall='C:\dev\' %}
+{% assign localappdata='%LOCALAPPDATA%\' %}
+{% assign appdata='%APPDATA%\' %}
+{% assign unzip='Extract-Archive' %}
+{% assign path='C:\dev\' %}
+{% assign prompt-start='C:\>' %}
+{% assign prompt=path | append: '>' %}
+{% assign terminal='PowerShell' %}
+{% assign rm = 'Remove-Item -Recurse -Force -Path ' %}
+{% capture rm-sdk %}Remove-Item -Recurse -Force -Path '{{dirinstall}}flutter'{% endcapture %}
+{% capture dart-files %}
+{{localappdata}}.dartServer
+{{appdata}}.dart
+{{appdata}}.dart-tool
+{% endcapture %}
+{% capture rm-dart-files %}
+{{prompt}} {{rm}} {{localappdata}}.dartServer,{{appdata}}.dart,{{appdata}}.dart-tool
+{% endcapture %}
+{% capture flutter-files %}{{appdata}}.flutter-devtools{% endcapture %}
+{% capture rm-flutter-files %}
+{{prompt}} {{rm}} {{flutter-files}}
+{% endcapture %}
+{% capture rm-pub-dir %}
+{{prompt}} {{rm}} {{localappdata}}Pub\Cache
+{% endcapture %}
+{% else -%}
+{% assign dirinstall='~/development' %}
+{% assign dirconfig='~/' %}
+{% assign path='~/development/' %}
```

Comment:
```suggestion
{% assign path='~/path/to/' %}
```

Reply:
**comment:** This path would be consistent with the install path in the Install Guides. I could change it there, too. Can we mark this as an issue to resolve separately?

Reply:
Can fix fix this as part of this PR? We shouldn't be specifying `/development/` as the directory you install the SDK in. I install my in `~/code/sdks` so if I followed these instructions I would be doing the wrong thing.

Reply:
Tony, please ignore my emoji's on your comment, but I see now way to delete them. I meant to thumbs up John's comment.

Reply:
(You can click the emojis again to delete them)